index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles/style.css">
+    <title>Rock, paper or scissors</title>
+
+</head>
+
+<body>
+
+    <main>
+        <section id="main-section" class="main-section">
+
+            <h1 class="title">Rock, Paper, Scissors!</h1>
+
+            <label>
+                <input type="radio" class="rock" name="throw" value="rock">
+
+                <img src="./assets/rock.png">
+
+            </label>
+
+            <label>
+                <input type="radio" class="paper" name="throw" value="paper">
+
+                <img src="./assets/paper.png">
+
+            </label>
+
+            <label>
+                <input type="radio" class="scissors" name="throw" value="scissors">
+
+                <img src="./assets/scissors.png">
+                
+            </label>
+
+            <div class="guess">
+
+                <button id="guess-button">
+
+                    Guess!!
+
+                </button>
+
+
+            </div>
+
+            <div class="current-throw" id="current-throw">
+
+            </div>
+
+            <div class="game-updates">
+
+                <div id="win">
```

Comment:
I think it's best to put the id on just the span. That way the textContent doesn't replace "win: " :)

Reply:
thanks for the feed back, we actually ended up doing that lol